src/vos/vos_pool.c

```diff
@@ -1010,6 +1016,17 @@ vos_pool_ctl(daos_handle_t poh, enum vos_pool_opc opc)
 	case VOS_PO_CTL_RESET_GC:
 		memset(&pool->vp_gc_stat, 0, sizeof(pool->vp_gc_stat));
 		break;
+	case VOS_PO_CTL_SET_POLICY:
+		if (param == NULL)
+			return -DER_INVAL;
+
+		p = param;
+		pool->vp_policy_desc.policy = p->policy;
+
+		for (i = 0; i < DAOS_MEDIA_POLICY_PARAMS_MAX; i++)
+			pool->vp_policy_desc.params[i] = p->params[i];
+
+		break;
```

Comment:
nit: unnecessary break on last switch case

Reply:
maybe better keep it in case a new opc added, and we missed it.